src/integration-tests/bash/archive.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-# Copyright 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 
 #
 # archive.sh <source_dir> <target_dir>
-#   - internal helper method called by run.sh
```

Comment:
Is this helper script used anywhere else except for the run.sh that is removed in this PR? If not, shall we remove this helper script as well?

Reply:
The archive.sh is not only called by run.sh, it is also called by other test scripts.

Reply:
Thanks for checking.